src/nupic/algorithms/backtracking_tm_cpp.py

```diff
@@ -713,77 +649,43 @@ def getSegmentInfo(self, collectActiveData = False):
             distSegSizes, distNSegsPerCell, distPermValues, distAges)
 
 
-  def getActiveSegment(self, c,i, timeStep):
-    """ For a given cell, return the segment with the strongest _connected_
-    activation, i.e. sum up the activations of the connected synapses of the
-    segments only. That is, a segment is active only if it has enough connected
-    synapses.
-    """
-
-    # TODO: add C++ accessor to implement this
-    assert False
-
-
-  def _getBestMatchingCell(self, c, timeStep, learnState = False):
-    """Find weakly activated cell in column. Returns index and segment of most
-    activated segment above minThreshold.
-    """
-
-    # TODO: add C++ accessor to implement this
-    assert False
-
-
-  def getLeastAllocatedCell(self, c):
-    """For the given column, return the cell with the fewest number of
-    segments."""
-
-    # TODO: add C++ accessor to implement this or implement our own variation
-    assert False
-
```

Comment:
I removed these three functions. They were never used, contained no logic, and did not seem to override anything.